### DISCUSSION TITLE
Model/Channel Login

### DISCUSSION THREAD

**Discussion by fl0wchart** [EchterAlsFake/unofficial-api-for-pornhub#48] [Ideas]
Is supporting model accounts and fetching data planned or even wanted in this repo?

**Comment by Egsagon** [comment]
If you mean public account data, its already available (user.bio and user.info, even if it is quite not fully implemented).

If you mean private account data where you need to login, it is not implemented since i do not own a model account and cannot do tests.

**Reply by fl0wchart** [reply]
Yes I mean private account data. I guess I could assist here if needed

**Reply by EchterAlsFake** [reply]
We would highly appreciate any help. Egsagon and I don't have much time currently, so feel free to fill your fork with some content :D

**Reply by Egsagon** [reply]
If you'd like we can also switch conv on discord.

Few preliminary questions:
- Does login work out of the box?
- What data do you want to retrieve? What page(s)/api call is it located?

Also having seen your 2FA implementation, you did remove some PHUB logic (account object connection, playlist, etc.) make sure you used latest commit.

**Reply by EchterAlsFake** [reply]
So I just created a server for Discord:

https://discord.gg/XmFwys8C
